### ISSUE TITLE
the process.env.NODE_ENV is undefined when i npm run build

### ISSUE THREAD

**Issue by FJHou** [SimulatedGREG/electron-vue#1008]
#### Found an issue or bug with electron-vue? Tell me all about it!
Questions regarding how to use `electron` or `vue` are likely to be closed as they are not direct issues with this boilerplate. Please seek solutions from official documentation or their respective communities. 

##### Describe the issue / bug.
\#
the process.env.NODE_ENV is undefined when i npm run build.

##### How can I reproduce this problem?
\#
electron-vue cli webpack.DefinePlugin also not work when npm run dev.

##### If visual, provide a screenshot.
\#
![electron](https://user-images.githubusercontent.com/22517598/80087769-23c22e00-858e-11ea-9ea8-d9b3f563cfc2.png)

##### Tell me about your development environment.
  * Node version: 10.16.1
  * NPM version: 6.13.4
  * vue-cli version: (if necessary)
  * Operating System: windows 10 64
#### *If you are looking to suggest an enhancement or feature, then feel free to remove everything above.*

**Comment by loama** [comment]
Also happening in MacOS, any fixes?

**Comment by loama** [comment]
https://auth0.com/blog/electron-tutorial-building-modern-desktop-apps-with-vue-js/ here is a good guide for anyone facing this issue

**Comment by FJHou** [comment]
> https://auth0.com/blog/electron-tutorial-building-modern-desktop-apps-with-vue-js/ here is a good guide for anyone facing this issue

Do u fixed it?

**Comment by loama** [comment]
Yeah it works @FJHou, I just didn’t make a PR because I am totally uncertain if that breaks other things.
